modules/vaos/app/services/vaos/systems_service.rb

```diff
@@ -92,6 +92,15 @@ def get_facility_available_appointments(facility_id, start_date, end_date, clini
       end
     end
 
+    def get_system_pact(system_id)
+      with_monitoring do
+        url = "/var/VeteranAppointmentRequestService/v4/rest/direct-scheduling/site/#{system_id}" \
+                "/patient/ICN/#{@user.icn}/pact-team"
+        response = perform(:get, url, nil, headers(@user))
```

Comment:
Totally a nitpick, but I think elsewhere user is attr_reader, and therefore we call the method `user` instead of instance variable.